cosmos/profiles/__init__.py

```diff
@@ -21,6 +22,7 @@
 from .trino.ldap import TrinoLDAPProfileMapping
 
 profile_mappings: list[Type[BaseProfileMapping]] = [
+    AthenaAccessKeyProfileMapping,
```

Comment:
Hey, should this have been included in the __all__ export on line 60?

Reply:
Yeah, good spot.

I have an upcoming [PR](https://github.com/astronomer/astronomer-cosmos/issues/609) for the Athena profile mapping, so I'll probably squeeze the fix in there. 

This shouldn't affect anyone, unless they're doing `from cosmos.profiles import *`